### PR TITLE
Add configuration option for passing blob directory path

### PR DESCRIPTION
**Pull request by FarooqAlaulddin**
## PL
To address #15. 

     -b, --blobs        -- Directory for Blobs. By default, it will create
                           a blobs directory at <home>/var/blobs unless a 
                           path is provided -b path/to/blobs
- argparse is used to handle argv.
- `-b, --blobs` is optional.

## Tests                
3 new test units were added:
- test_create_folders_and_files_w_blobs
- test_zeo_conf_content_w_default_blobs
- test_zeo_conf_content_w_specified_blobs

Other unit tests edited to reflect the change. 

Test Status:
- Ran 27 tests in 0.052s - OK image

### CODE REVIEW

File: src/zope/mkzeoinstance/tests/test_unix.py
Comment (dataflake) [inline src/zope/mkzeoinstance/tests/test_unix.py]: While this may work with some packages it is not the canonical way unit tests are run for zopefoundation packages, we use `zope.testrunner` in conjunction with `tox` by default. Please remove this comment.

**Comment by dataflake** [comment]
Thank you for you contribution. We require all contributors to sign the Zope contributor agreement before we can merge the code, please see https://www.zope.dev/developer/becoming-a-committer.html

File: src/zope/mkzeoinstance/tests/test_unix.py
Comment (FarooqAlaulddin) [reply src/zope/mkzeoinstance/tests/test_unix.py]: comment removed.

**Comment by FarooqAlaulddin** [comment]
> Thank you for you contribution. We require all contributors to sign the Zope contributor agreement before we can merge the code, please see https://www.zope.dev/developer/becoming-a-committer.html

Done.

**Comment by icemac** [comment]
@FarooqAlaulddin Could you please run the tests locally using `tox -pauto`?
There are some linting tests which fail and the code does not run on Python < 3.9.
(Sorry GitHub Actions was disabled, so the failures do not yet show up here. But for your next commit they will run again.)

File: src/zope/mkzeoinstance/tests/test_unix.py
Comment (dataflake) [reply src/zope/mkzeoinstance/tests/test_unix.py]: This comment is still in the PR

**Review by dataflake — CHANGES_REQUESTED** [review]
Once the comment is removed this looks good, thank you.

**Review by dataflake — APPROVED** [review]

**Comment by FarooqAlaulddin** [comment]
Great!

**Comment by dataflake** [comment]
@FarooqAlaulddin Once a PR is ready for merging (all issues fixed, approvals given) the author can do the merge, so just go ahead and do so.

**Comment by icemac** [comment]
Just released as https://pypi.org/project/zope.mkzeoinstance/5.1/.

**Comment by icemac** [comment]
This PR seems to cause https://github.com/plone/plone.recipe.zeoserver/pull/45#issuecomment-1531537732

**Comment by petschki** [comment]
I've fixed that here #18

**Pull request merged**
